### PR TITLE
add uidResolver to docfx_document_schema.md 

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: Documentation/spec/docfx_document_schema.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ It defines how applications interpret the property. If not defined, the behavior
 | Value      | Description
 |------------|-------------
 | `default`  | It means that no interpretion will be done to the property.
-| `uid`      | `type` MUST be `string`. It means the property defines a unique identifier inside current document model
+| `uid`      | `type` MUST be `string`. With this value, the property name MUST be `uid`. It means the property defines a unique identifier inside current document model
```
Comment (superyyrrzz) [inline Documentation/spec/docfx_document_schema.md]: >model [](start = 164, length = 5)

append `.`

File: Documentation/spec/docfx_document_schema.md [hunk 2]
```diff
@@ -255,3 +255,13 @@ Here's the schema to describe these operations:
             2. Little chance that keywords DocFX defines duplicate with what JSON schema defines, after all, JSON schema defines a finite set of reserved keywords.
             3. For example[Swagger spec](http://swagger.io/) is also based on JSON schema and the fields it introduces in has no prefix. 
     * Decision: *Remove* `d-` prefix.
+3. What's remaining work if to apply schema to the complex data model, for example, ManagedReference, or UniversalReference?
+    * 1. OPS plugin framework, to insert metadata into the data model, for example, git commit id, git contributers.
+      Solution: A TagInterpreter plugin framework to insert metadata if the property contains `metadata` tag
```
Comment (superyyrrzz) [inline Documentation/spec/docfx_document_schema.md]: The same tag may have different behavior in different schema. Is this supported? e.g. gitcomit's definition is different between conceptual and mref

Comment (vicancy) [inline Documentation/spec/docfx_document_schema.md]: No its not, use another tag name instead

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**